index.Rmd

```diff
@@ -295,18 +295,19 @@ plot1=ggplot(data = All_groupings,aes(x=Year,y=percpos,color=Pathogen)) +
         ylab("Pathogen Prevalence (%)")+
         scale_x_continuous(breaks=c(2008:2019))
 ```
-
+ 
```

Comment:
Instead of using the default ggplot theme, you can check out [](https://ggplot2.tidyverse.org/reference/ggtheme.html) to see different theme types that could make your graph stand out more.